src/css/SharedStyling.js

```diff
@@ -78,11 +78,13 @@ export const Content = styled.div`
   @media ${device.laptop} {
     padding: ${(props) =>
       props.padding || GLOBALS.ADJUSTMENTS.MARGIN.DEFAULT.TABLET};
+    // max-width: ${(props) => props.maxWidth || "1213px"};
   }
 
   @media ${device.mobileM} {
     padding: ${(props) =>
       props.padding || GLOBALS.ADJUSTMENTS.MARGIN.DEFAULT.MOBILE};
+    max-width: 100%;
```

Comment:
Should be `max-width: ${(props) => props.maxWidth || "100%"};` since we should leave a way to override if the default is not needed